input/fsh/SD_Genomics.fsh

```diff
@@ -235,20 +235,22 @@ Description:    "The area of the genome region referenced in testing for variant
 * component[genomicReferenceSequenceId] ^definition = "Range(s) of DNA sequence examined. The genomic reference sequence is a contiguous stretch of chromosome DNA that spans all of the exons of the gene and includes transcribed and non transcribed stretches. For this ID use either the NCBI genomic nucleotide RefSeq IDs with their version number (see: NCBI.NLM.NIH.Gov/RefSeq) or use the LRG identifiers, without transcript (t or p) extensions -- when they become available. (source: LOINC)"
 * component[genomicReferenceSequenceId].code = LNC#48013-7
 
-// * insert ReduceText
-// * insert ReduceText(referenceRange)
-// * insert ReduceText(component)
-// These are the additional MS on top of US Core Lab Observation
 * component and component.code and component.value[x] and component.dataAbsentReason MS
 
+Profile:    GenomicDiagnosticImplication
+Parent:     USCoreObservationLab
+Id: mcode-genomic-diagnostic-implication
+Title:      "Genomic Diagnostic Implication"
+Description:    "Observation stating a linkage between one or more genotype/haplotype/variation Observations and evidence for or against a particular disease, condition, or cancer diagnosis."
+* code = LNC#81259-4 "Associated phenotype" //"Associated phenotype"
+* derivedFrom only Reference(CancerGeneticVariant)
+* value[x] 0..0 
+* value[x] ^definition = "Not used in this profile. MustSupport is inherited from the parent profile (USCoreObservationLab) and should be ignored by implementers for this element."
+* insert ObservationComponentSlicingRules
+* insert CreateComponent(clinicalSignificance, 0, 1)
 
-//* identifier ^slicing.discriminator.type = #value
-//* identifier ^slicing.discriminator.path = "type.coding.code"
-//* identifier ^slicing.rules = #open
-//* identifier contains
-//    AccessionIdentifier 0..1 MS and
-//    FillerOrderNumber 0..1 MS and
-//    PlacerOrderNumber 0..1 MS
-//* identifier[AccessionIdentifier].type = IDTYPE#ACSN
-//* identifier[FillerOrderNumber].type = IDTYPE#FILL
-//* identifier[PlacerOrderNumber].type = IDTYPE#PLAC
\ No newline at end of file
+* component[clinicalSignificance].code = LNC#53037-8
```

Comment:
- Why do you model this as a component rather than the value of the Observation (assuming an Observation is used)? 
- Is it correct there is only one clinical significance value for each CancerGeneticVariant?
- Should LNC#53037-8 be the code, rather than the component code?

Reply:
- re: modeling as a component than a value: _Per the CG Reporting IG, clinical significance is not modeled as an observation of its own but a set of qualifiers of a greater diagnostic implication. It's part of an overarching design approach in this IG of handling report implications overall and is applied to not just diagnostic implications (which focus more on phenotypes) but also therapeutic implications._
- re: one clinical significance for CancerGeneticVariant: _yes one per variant at least with the searches I've done in  ClinVar and also noted in the CG Reporting IG. Not sure of other genomics knowledgebases express it differently. I can check if it's a problem._
- Re: Should LNC#53037-8 be the code, rather than the component code: _Answered in first bullet point. Not per the general design paradigm of implications handled in the CG Reporting IG. clinical-significance is part of a greater set of an evaluation of diagnostic implications under one profile._